db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
I even remember there were a time when these items wasn't enabling skill usage, and somebody reported it as an issue and then was added the enable skill script to those items.

I just don't remember where was reported, but if needed I can search for it.